src/jit/flowgraph.cpp

```diff
@@ -3189,9 +3189,15 @@ void Compiler::fgComputePreds()
         if (ehDsc->HasFilter())
         {
             ehDsc->ebdFilter->bbFlags |= BBF_JMP_TARGET | BBF_HAS_LABEL;
+
+            // The first block of a filter has an artifical extra refcount.
+            ehDsc->ebdFilter->bbRefs++;
```

Comment:
Thanks - so I gather that this is not something that is new, but was previously omitted. I was trying to figure out how that was a new requirement.